tweets/models.py

```diff
@@ -7,5 +7,17 @@ class Tweet(models.Model):
     content = models.TextField(max_length=150)
     created_at = models.DateTimeField(auto_now_add=True)
 
+    @property
+    def liked_count(self):
+        return self.likes.count

```

Comment:
```suggestion
    @property
    def liked_count(self):
        return self.likes.count()
```
countはメソッドなので、実行してあげないと実数値返してくれないです。

templateで使えるのテンプレートエンジンのおかげなので、propertyとして使うならメソッド実行させる必要があります。こうすれば、JsonResponseのcontextのところできちんとJSON変換できるようになります。

templateでは、count()でなく、countで表示される理由
これは単純で、テンプレート内で関数実行はできないからですね。代わりに、テンプレートではcountとすることで、count()メソッドを実行させるようになってます。試しに、template内で{{ tweet.likes.count() }}としてみてレンダリングしてみてください。TemplateSyntaxErrorが出ると思います